tests/kdc/check-kadmin.in

```diff
@@ -59,7 +59,7 @@ kinit="${kinit} -c $cache ${afs_no_afslog}"
 kgetcred="${kgetcred} -c $cache"
 kdestroy="${kdestroy} -c $cache ${afs_no_unlog}"
 
-foopassword="foo"
+foopassword="foobar"
```

Comment:
Can we have a check that simple passwords are rejected?

Reply:
Also, let's future-proof this password, make it something like `fooLongPasswordYo123;` :)